Classes/Emogrifier.php

```diff
@@ -1501,4 +1505,34 @@ private function getNodesToExclude(\DOMXPath $xPath)
 
         return $excludedNodes;
     }
+
+    /**
+     * Handle warnings in process() method
+     *
+     * @param int $errNo
+     * @param string $errStr
+     * @param string $errFile
+     * @param int $errLine
+     * @param array $errContext
+     *
+     * @return bool
```

Comment:
We don't need a return value as this method throws an exception if there is a problem and does not throw it if everything is fine. So this can be return void, and the return false can go away.